bpf/lib/process.h

```diff
@@ -324,6 +337,40 @@ struct {
 	__type(value, struct execve_heap);
 } execve_heap SEC(".maps");
 
+/* the tg_execve_joined_info_map allows to join and combine
+ * exec info that is gathered during different hooks
+ * through the execve call.
```

Comment:
Please document the hooks and how information is gathered.

Reply:
Sure done